bucoffea/monojet/monojetProcessor.py

```diff
@@ -527,6 +530,16 @@ def ezfill(name, **kwargs):
             ezfill('dphijm',             dphi=df["minDPhiJetMet"][mask],    weight=region_weights.weight()[mask] )
             ezfill('dphijr',             dphi=df["minDPhiJetRecoil"][mask],    weight=region_weights.weight()[mask] )
 
+            # Photon CR data-driven QCD estimate
+            if df['is_data'] and re.match("cr_g.*", region) and re.match("(SinglePhoton|EGamma).*", dataset):
+                w_imp = photon_impurity_weights(photons[leadphoton_index].pt, df["year"])
+                output['recoil'].fill(
+                                    dataset=data_driven_qcd_dataset(dataset),
+                                    region=region,
+                                    mjj=df["recoil_pt"][mask],
```

Comment:
The only point I'm confused is in this line actually, I'm not sure why you are filling an axis named "mjj" with "recoil_pt". Does this histogram have a mjj axis, or should it be something like 
'''
recoil=df["recoil_pt"][mask]
'''

Reply:
Thanks for catching that. Indeed, it should have been `recoil=...`